### PR TITLE
Fix Transpose using simd

### PR DESCRIPTION
**Pull request by nilsdeppe**
## Proposed changes

@knelli2 @isaaclegred could you test if this works for you.

It's a 2 character change that actually fixes the issue...

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Review by knelli2 — APPROVED** [review]
Tested locally and on wheeler and the single BH executable with default input file doesn't seg fault any more. Haven't done other tests though. We should wait for @isaaclegred to do his test before we merge

**Comment by isaaclegred** [comment]
No segfault on my end either!  Running with a TOVstar that previously failed within the first step

**Review by isaaclegred — APPROVED** [review]

**Review by wthrowe — APPROVED** [review]

**Pull request merged**
